package/network/utils/comgt/files/ncm.sh

```diff
@@ -126,18 +137,27 @@ proto_ncm_setup() {
 		json_select ..
 	}
 
-	json_get_vars connect
-	eval COMMAND="$connect" gcom -d "$device" -s /etc/gcom/runcommand.gcom || {
-		echo "Failed to connect"
-		proto_notify_error "$interface" CONNECT_FAILED
-		return 1
-	}
+#	json_get_vars connect
+#	eval COMMAND="$connect" gcom -d "$device" -s /etc/gcom/runcommand.gcom || {
+#		echo "Failed to connect"
+#		proto_notify_error "$interface" CONNECT_FAILED
+#		return 1
+#	}
 
```

Comment:
I don't yet see what is the need to move this block? Your change shouldn't have any effect.